solutions/bio1/DistinctPrimeFactorisation/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { SCodeBlock, SText, STitle, SCode } from "components";
+
+export const DistinctPrimeFactorisation = () => {
+  return (
+    <>
+      <STitle>Idea</STitle>
+      <SText>
+        Since $n$ is small, we can actually just brute force 
+        all numbers from $1$ to $n$ and check if they are factors 
+        of $n$. First, set <SCode>answer=1</SCode>. Now, for each 
+        number from $1$ to $n$, if the number is a factor of $n$, 
+        we can first multiply <SCode>answer</SCode> by the number,
+        then divide $n$ by the number until it is no longer a factor
+        of $n$.
+      </SText>
+      <SText>
+        Of course, this seems far too simple to be correct. However,
+        here is a proof:
+      </SText>
+      <SText>
+        Let $p$ be a prime factor of $n$. Once we have removed 
+        all multiples of $p$ from $n$, we will never encounter
+        $p$, or a lower prime, again. This is because if we did,
+        it would mean we had not removed all multiples of $p$ from
```

Comment:
This proof is phrased quite weird imo

I think it's a lot clearer to say:
- After the $i$-th iteration, $n$ is no longer divisible by any number $d \in [2, i]$.
- Therefore, if $n$ is divisible by $i$ at the start of the $i$-th iteration, $i$ cannot be divisible by any $d \in [2, i)$ otherwise $n$ is also divisible by $d$ causing a contradiction, so $i$ must be prime.

...or similar.

Reply:
yeah sorry I'll change it - I'm about to push another problem, then i'll edit this one :)